src/components/Icon.stories.tsx

```diff
@@ -75,6 +75,10 @@ export const Icon = (props: IconProps) => {
     "arrowRight",
     "menuClose",
     "menuOpen",
+    "bxArrowFromBottom",
+    "bxArrowFromLeft",
+    "bxArrowFromRight",
+    "bxArrowFromTop",
```

Comment:
(giving the same suggestion here)
```suggestion
    "arrowFromBottom",
    "arrowFromLeft",
    "arrowFromRight",
    "arrowFromTop",
```